README.md

```diff
@@ -19,15 +19,11 @@ You may use **any resource** other than each other to complete this diagnostic.
 This includes referencing talk materials, appropriate documentation, and
 searching for help online.
 
```

Comment:
Try to include changes like this in the previous commit next time. I consider things like this part of updating from template.